config.go

```diff
@@ -39,6 +39,7 @@ type eureka struct {
 	PreferSameZone        bool     // default false
 	RegisterWithEureka    bool     // default false
 	Retries               int      // default 3
+	ServerContext         string   // default "eureka/v2"
```

Comment:
- [ ] "Context" is an odd word here.  
  How about "ServerURLBase?"
- [ ] Move this line up to follow the "ServerPort" field at line 36 above.

Reply:
Agree, changed ServerContext to ServerURLBase everywhere.
P.S. Got 'context' name from official Eureka Wiki, but 'URLBase' is more clear for a client.